packages/oak-addon-basic-components/lib/core/Editor/ColorButton.js

```diff
@@ -75,9 +75,23 @@ export default ({ className }) => {
       <DropdownMenu>
         <ColorField
           ref={colorFieldRef}
-          className="oak-color-field py-3"
+          className="py-3"
           value={color}
           onChange={onChange}
+          popperOptions={{
+            strategy: 'relative',
+            modifiers: [{
+              name: 'computeStyles',
+              options: {
+                gpuAcceleration: false,
+                adaptive: false,
+                roundOffsets: ({ y }) => ({
+                  x: 0,
+                  y: Math.round(y - 30),
```

Comment:
if it's just for having offset you can simply use offset option I think https://popper.js.org/docs/v2/modifiers/offset/